python-avd/pyavd/_cv/workflows/models.py

```diff
@@ -73,6 +73,23 @@ class CVPathfinderMetadata:
     device: CVDevice | None = None
 
 
+@dataclass
+class CVWSBuildConfigValidationError:
```

Comment:
We prefer not to have abbreviated variables. Also consider if this class is always an error or could it be a warning?
```suggestion
class CVWorkspaceBuildConfigValidationError:
```

Reply:
Fixed